data/integrations/integrations.yml

```diff
@@ -2,3 +2,6 @@
 - name: google
   available_versions:
     - google/2021-09-21
+- name: shopify
+  available_versions:
+    - shopify/2022-02
```

Comment:
We should leave this entry for https://github.com/Shopify/product-taxonomy/pull/127 which adds this mapping

Reply:
removed.